app/src/main/java/cm/aptoide/pt/home/apps/AppsFragment.java

```diff
@@ -219,6 +246,30 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         .map(appClick -> appClick.getApp());
   }
 
+  @Override public Observable<App> resumeAppcUpgrade() {
+    return appItemClicks.filter(
```

Comment:
Like i said before, maybe the resume, pause, cancel etc methods could be merged to the resumeUpdate pauseUpdate etc. I understand your point that since they call different set pausing state methods, but maybe we could create a wrapper of that with the action and with the App, or just with a boolean and the App (to be more clean) and then get that wrapper and call the correct setstandbystate method